components/gsma-authenticators/src/main/java/com/wso2telco/gsma/authenticators/voice/VoiceIVRFutureCallback.java

```diff
@@ -0,0 +1,88 @@
+package com.wso2telco.gsma.authenticators.voice;
+
+import com.wso2telco.gsma.authenticators.AuthenticatorException;
+import com.wso2telco.gsma.authenticators.Constants;
+import com.wso2telco.gsma.authenticators.DBUtils;
+import com.wso2telco.gsma.authenticators.util.BasicFutureCallback;
+import com.wso2telco.ids.datapublisher.model.UserStatus;
+import com.wso2telco.ids.datapublisher.util.DataPublisherUtil;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.apache.http.HttpResponse;
+import org.apache.http.util.EntityUtils;
+import org.codehaus.jettison.json.JSONException;
+import org.codehaus.jettison.json.JSONObject;
+import org.wso2.carbon.identity.application.authentication.framework.context.AuthenticationContext;
+
+import java.io.IOException;
+import java.sql.SQLException;
+
+/**
+ * Created by sheshan on 5/24/17.
+ */
+public class VoiceIVRFutureCallback extends BasicFutureCallback {
+
+    private static Log log = LogFactory.getLog(VoiceIVRFutureCallback.class);
+    private static final String IS_FLOW_COMPLETED = "isFlowCompleted";
+    private final String UNKNOWN_USER = "UNKNOWN_USER";
+    private final String ACTIVE = "ACTIVE";
+    private final String ERROR = "ERROR";
+    private final String AUTHENTICATED = "AUTHENTICATED";
+    private AuthenticationContext authenticationContext;
+    private String msisdn;
+
+    private String SEND_IVR = "ivr_sent";
+    private String SEND_IVR_FAIL = "ivr_falied";
+
+    VoiceIVRFutureCallback() {
+    }
+
+    VoiceIVRFutureCallback(AuthenticationContext authenticationContext , String msisdn) {
+        this.authenticationContext = authenticationContext;
+        this.msisdn = msisdn;
+    }
+
+    public void completed(HttpResponse response) {
+        if ((response.getStatusLine().getStatusCode() == 200)) {
+            String json = null;
+            try {
+                json = EntityUtils.toString(response.getEntity());
+                JSONObject jsonObj = new JSONObject(json);
+                log.info("~~~~~~~~~~~~~~~############!!!!!!!!~~~~~~~#########!!!!!!!!!");
+                log.info(json);
+                log.info("~~~~~~~~~~~~~~~############!!!!!!!!~~~~~~~#########!!!!!!!!!");
+                //"outcome":"ERROR"
+                // NOT_AUTHENTICATED
+                String outCome = jsonObj.getString("outcome");
+//                log.info("****************************************************8*****"+outCome);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.wso2telco.com:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)